documentation/src/main/asciidoc/topics/upgrading.adoc

```diff
@@ -25,13 +25,14 @@ The following changes apply to the Infinispan 13 configuration schemas:
 
 * The `urn:infinispan:config:store:soft-index` namespace is no longer available.
 * The `file-store` element in the `urn:infinispan:config` namespace now defaults to soft-index file stores.
-* The `single-file-store` element is added to the `urn:infinispan:config` namespace if you want to configure a single file store, which is deprecated.
+* The `single-file-store` element has been added to the `urn:infinispan:config` namespace if you want to configure a single file store, which is deprecated.
+* The `endpoints` element in the `urn:infinispan:server` namespace is no longer a repeating element, but a wrapper for 0 or more `endpoint` elements.
 
 === Migrating Soft-Index File Stores
 
 ==== Declarative
 
-If you are upgrading to Infinispan 13 from a previous version and your caches include any configuration with the `soft-index-file-store` element, you should convert that configuration to use the `file-store` element instead.
+If you are upgrading to Infinispan 13 from a previous version, and your caches include any configuration with the `soft-index-file-store` element, you should convert that configuration to use the `file-store` element instead.
```

Comment:
Are these changes completely compatible with the old Cache JSON format? Or do we need to document some incompatibilities here?

Reply:
AFAICT we are compatible.